manager/manager.go

```diff
@@ -218,7 +221,7 @@ func New(config *Config) (*Manager, error) {
 	m := &Manager{
 		config:      config,
 		listeners:   listeners,
-		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig),
+		caserver:    ca.NewServer(RaftNode.MemoryStore(), config.SecurityConfig, caReconciliationRetryInterval),
```

Comment:
Not convinced that this should be a param. Why not a constant? Seems like we're polluting a function with one more variable just so we can test it easily.